include/liboauthcpp/liboauthcpp.h

```diff
@@ -39,40 +39,49 @@ void SetLogLevel(LogLevel lvl);
 /** Deprecated. Complete percent encoding of URLs. Equivalent to
  *  PercentEncode.
  */
-std::string URLEncode(const std::string& decoded);
+std::string URLEncode(const std::string& decoded,
+                      bool rfc3986Compliant = false);

```

Comment:
If this is effectively a bug fix, I don't think we need to have this flag at all -- we should just correct the issue.